doc/examples.rst

```diff
@@ -1,41 +1,23 @@
-Advanced tutorials
+Examples
 ==================
 
-Advanced features
------------------
-
-.. toctree::
-  :maxdepth: 1
-
-  notebooks/Batch mode overview
-
 Celestial mechanics and astrodynamics
 -------------------------------------
 
 .. toctree::
   :maxdepth: 1
 
   notebooks/The restricted three-body problem
+  notebooks/Poincaré sections
```

Comment:
I don't think the Poincare' sections tutorial belongs here. Perhaps in the event detection tutorials?